package.json

```diff
@@ -55,28 +55,28 @@
     "@dash4/plugin-npm-scripts": "0.9.3",
     "@dash4/plugin-readme": "0.9.3",
     "@dash4/server": "0.9.3",
-    "@namics/prettier-config": "2.0.0",
-    "@namics/ts-config": "1.1.0",
-    "acorn": "8.0.4",
+    "@namics/prettier-config": "2.0.1",
+    "@namics/ts-config": "1.1.1",
+    "acorn": "8.2.4",
     "eslint": "7.16.0",
     "eslint-plugin-import": "2.22.1",
     "node-fetch": "2.6.1",
-    "npm-check-updates": "10.2.5",
+    "npm-check-updates": "11.5.13",
     "npm-run-all": "4.1.5",
-    "prettier": "2.2.1",
-    "typescript": "4.1.3"
+    "prettier": "2.3.0",
+    "typescript": "4.2.4"
   },
   "peerDependencies": {
-    "eslint": ">=6",
+    "eslint": ">=7",
```

Comment:
wouldn't change this, if not really needed

Reply:
Downgraded.